src/backend/marsha/core/serializers/base.py

```diff
@@ -27,6 +31,26 @@
 KEY_REGEX = re.compile(KEY_PATTERN)
 
 
+def get_video_cloudfront_url_params(video_id):
+    """
+    Generate the policy and sign it to allow to access to all resources for a given video id.
+    """
+    resource = (
+        f"{settings.AWS_S3_URL_PROTOCOL}://{settings.CLOUDFRONT_DOMAIN}/{video_id}/*"
+    )
+    cache_key = f"cloudfront_signed_url|video|{video_id}|{resource}"
```

Comment:
NIT: Good point for the prefix, I would suggest to separate "namespaces" with colon (`:`) as it is a common convention since Redis doesn't provide namespaces. I don't have much sources for this practice, just found https://www.oreilly.com/library/view/redis-essentials/9781784392451/ch06s03.html but I know it's quite common.